de.metas.adempiere.adempiere/base/src/main/java/de/metas/report/ExecuteReportStrategy.java

```diff
@@ -58,7 +58,7 @@ public static ExecuteReportResult of(@NonNull final OutputType outputType, @NonN
 
 		public static ExecuteReportResult of(@NonNull final String filename, @NonNull final OutputType outputType, @NonNull final byte[] reportData)
 		{
-			return new ExecuteReportResult(null, outputType, reportData);
+			return new ExecuteReportResult(filename, outputType, reportData);
```

Comment:
yeah, that's the last fix that i made in x-mas and didn't yet put to master

Reply:
yeah, thought it's good to put it with this ocassion!